src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -78,7 +78,7 @@ public SyntaxTree SyntaxTree
             {
                 return GetOperationCore(node, cancellationToken);
             }
-            catch (Exception e) when (FatalError.ReportAndCatchUnlessCanceled(e))
+            catch (Exception e) when (FatalError.ReportAndCatchUnlessCanceled(e, cancellationToken))
```

Comment:
👍 This one seems like a good change (could only impact telemetry in release builds).